Dockerfile

```diff
@@ -3,17 +3,15 @@ FROM centos:7
 # Install dependencies
 RUN yum -y install epel-release  && \
     yum -y install bash wget unzip \
-           pexpect python-daemon  bubblewrap gcc \
-           bzip2  openssh openssh-clients python2-psutil\
-           python36 python36-devel python36-setuptools\
-           nginx supervisor && \
-           localedef -c -i en_US -f UTF-8 en_US.UTF-8
-RUN easy_install-3.6 -d /usr/lib/python3.6/site-packages pip && \
-    ln -s /usr/lib/python3.6/site-packages/pip3 /usr/local/bin/pip3
+    pexpect python-daemon  bubblewrap gcc \
+    bzip2  openssh openssh-clients python2-psutil\
+    python36 python36-devel python36-setuptools\
+    nginx supervisor && \
+    localedef -c -i en_US -f UTF-8 en_US.UTF-8
 
-RUN /usr/local/bin/pip3 install ansible cryptography docutils psutil PyYAML \
-                 pyOpenSSL flask flask-restful uwsgi netaddr notario && \
-    /usr/local/bin/pip3 install --no-cache-dir ansible-runner==1.3.2 && \
+RUN python3.6 -m pip install ansible cryptography docutils psutil PyYAML \
+    pyOpenSSL flask flask-restful uwsgi netaddr notario && \
```

Comment:
What is this indenting change for?

Reply:
Taking a look to the raw file you will see that the look is great :-)

Reply:
OK